developers.md

```diff
@@ -0,0 +1,23 @@
+# Signalen for developers and vendors
+
+The Signalen team plans to start work on making Signalen open and reusable during spring and summer 2020. We're currently working on Amsterdam's Signalen repo, with the goal of moving Signalen to a separate master repo later this year.
+
+Signalen is changing every day, but in the meantime we invite you to explore our code, have a tinker and give us feedback.
+
+You can learn more about our work or contribute to Signalen at:
+
+* [Amsterdam's Signalen Github repo](https://github.com/amsterdam/signals)
+* [Signalen API spec](https://api.data.amsterdam.nl/api/swagger/?url=/signals/swagger/openapi.yaml) and [API documentation](https://api.data.amsterdam.nl/signals/)
+
+Join our community at:
+
+* [Signalen channel on the Common Ground Slack](https://samenorganiseren.slack.com/archives/CQB3TNZGB)
+* [Signalen page on Common Ground](https://commonground.nl/groups/view/54477168/signalen)
+
+## Vendors
+
+We will need vendors to both sell Signalen or sell services on top of Signalen in the future.
```

Comment:
Perhaps a clearer language about what we need them to sell? I imagine we don't need vendors to sell the source code, but to sell implementation, integration, hosting and maintenance.

Reply:
Great!